server/routes/link/RequestLinkController.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from 'express'
+import RequestLinkView from './RequestLinkView'
+
+export default class RequestLinkController {
+  async getRequestLinkView(req: Request, res: Response) {
+    const view = new RequestLinkView(req.session?.requestLinkForm || {}, req.flash('errors'))
+
+    res.clearCookie('create_barcode_token').render('pages/link/requestLink', view.renderArgs)
```

Comment:
Could maybe use a unit test to check the cookie is cleared?

Reply:
👍 I've just written an integration test for this as part of the next part of dev - can I commit it in the next PR?

Reply:
yes next PR